Assets/Scripts/UI/Hangar/HangarNodeUI.cs

```diff
@@ -19,6 +19,7 @@ public enum HangarPanel
 
     public Slider fuelSlider;
     public FuelButton fuelButton;
+    private Button fuelButtonControl; 

```

Comment:
just access the button from FuelButton, no need to grab another variable

Reply:
ok, we do have to call getcomponent multiple times this way though

Reply:
make it grab a reference to itself on awake. one getcomponent call. access it with fuelButton.button